gdk/static/cli_model.json

```diff
@@ -90,6 +90,20 @@
                     "--bucket"
                 ],
                 "help": "Name of the s3 bucket to use for uploading component artifacts."
+            },
+            "region": {
+                "name": [
+                    "-r",
+                    "--region"
+                ],
+                "help": "Name of the AWS region to use component creation."
+            },
+            "options": {
+                "name": [
+                    "-o",
+                    "--options"
+                ],
+                "help": "Extra configuration options used during component version creation. This argument needs to be a valid json string or file path to a JSON file containing the publish options."
```

Comment:
We need to explain here that this will overwrite the publish options defined on the configuration file.